benchmarks/src/profiler.py

```diff
@@ -158,7 +150,7 @@ def _generate_function_call_code(self, func: Callable[[], T]) -> str:
 
 @contextmanager
 def profile_benchmark(
-    flame_config: FlameGraphConfig | None = None,
+    _flame_config: FlameGraphConfig | None = None,
```

Comment:
Systematically cleanup the benchmarks files for unused assignments, variables, parameters etc